<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Algoritmo2Window</class>
 <widget class="QWidget" name="Algoritmo2Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>460</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Algoritmo 2</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(210, 210, 210);</string>
  </property>
  <widget class="QFrame" name="cabeceraAlg2">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>0</y>
     <width>431</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(166, 166, 166);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="TituloCabecera">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>ALGORITMO 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="Texto">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>401</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>IMAGENES A BLANCO Y NEGRO</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="tiempo1textoAlg2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Tiempo 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="tiempo2textoAlg2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tiempo 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="tiempo5textoAlg2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tiempo 5</string>
   </property>
  </widget>
  <widget class="QLabel" name="tiempo4textoAlg2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tiempo 4</string>
   </property>
  </widget>
  <widget class="QLabel" name="tiempo3textoAlg2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tiempo 3</string>
   </property>
  </widget>
  <widget class="QLabel" name="tiempo4Alg2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>210</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(86, 93, 103);
border-color: rgb(255, 255, 255);
border-style: solid;
color: rgb(255, 255, 255);
border-width: 1px
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tiempo1Alg2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(86, 93, 103);
border-color: rgb(255, 255, 255);
border-style: solid;
color: rgb(255, 255, 255);
border-width: 1px
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tiempo2Alg2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(86, 93, 103);
border-color: rgb(255, 255, 255);
border-style: solid;
color: rgb(255, 255, 255);
border-width: 1px
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tiempo5Alg1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>250</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(86, 93, 103);
border-color: rgb(255, 255, 255);
border-style: solid;
color: rgb(255, 255, 255);
border-width: 1px
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tiempo3Alg2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(86, 93, 103);
border-color: rgb(255, 255, 255);
border-style: solid;
color: rgb(255, 255, 255);
border-width: 1px
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ExecuteAlg2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(234, 96, 96);
color: rgb(255, 255, 255);
border-color: rgb(234, 96, 96);


</string>
   </property>
   <property name="text">
    <string>EJECUTAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ResetAlg2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>400</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(234, 96, 96);
color: rgb(255, 255, 255);
border-color: rgb(234, 96, 96);


</string>
   </property>
   <property name="text">
    <string>RESETEAR</string>
   </property>
  </widget>
  <widget class="QLabel" name="tiempoMediotextoAlg2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>340</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Media</string>
   </property>
  </widget>
  <widget class="QLabel" name="tiempoMedioAlg1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>340</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(86, 93, 103);
border-color: rgb(255, 255, 255);
border-style: solid;
color: rgb(255, 255, 255);
border-width: 1px
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="SelectDirAlg2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(234, 96, 96);
color: rgb(255, 255, 255);
border-color: rgb(234, 96, 96);


</string>
   </property>
   <property name="text">
    <string>SELECCIONAR 
DIRECTORIO</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
